Automattic/jetpack
# Admin Page: Modularize Redux init into separate helper class

#### Changes proposed in this Pull Request:
Refactors the Redux initialization code into a staticized helper class.

This is in preparation for a new Jetpack Search wp-admin dashboard, which will re-use the existing Admin Page Redux code -- see #19810 as a working example. At the very least, this change improves code readability and fixes several linting errors.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* (These changes should not affect functionality.)
* Apply these changes to your Jetpack site. Ensure that the Admin Pages (both the Dashboard and the Settings pages) all work as expected.
* Additionally, check the browser console and ensure that the `window.Initial_State` object has been properly populated.
